repack-mysql.sh

```diff
@@ -58,8 +58,8 @@ $TAR -czf $OLDPWD/$RESOURCES/mysql-Linux-amd64.tar.gz \
   share/*.txt \
   share/charsets \
   share/english \
-  lib/libcrypto.* \
-  lib/libssl.* \
+  bin/libcrypto.* \
```

Comment:
This actually doesn't work, as these are symlinks in the original, and thus the produced tarball has dangling symlinks.

Reply:
Good catch, thanks!